src/BugsnagUnity/Delivery.cs

```diff
@@ -524,21 +524,54 @@ private IEnumerator ProcessCachedItems(Type t)
 
                     try
                     {
+                        // if something goes wrong at this stage then we silently discard the file as it's most likley an io error at some point
```

Comment:
Let's be a bit more specific about this:

```suggestion
                        // if something goes wrong at this stage then we silently discard the file as it's most likely that the file wasn't fully serialised to disk
```